workflow/drilling.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Drilling Into Your Data
+---
+
+The Drill Down feature enhances your ability to investigate, analyze, and resolve issues without needing to switch contexts. Instead of sifting through large volumes of data indiscriminately, focus your efforts on drilling into relevant and focused subsets of data in order to save your team time and effort.
+
+This feature is available across all of Middleware and works by passing context through from the beginning to the end of a request. Clicking on any of Middleware's interactive tables will produce popouts that divulge the interconnectedness of your system infrastructure in a single window. Middleware's interactive tables will produce popouts that allow you to drill down to root cause regardless of which direction you start from.
```

Comment:
This section flows a little weird and I think some of it is the wording. Here's a suggested edit.

"Clicking on any of Middleware's interactive tables will produce popouts that walk you through each component involved in a request, how that component is behaving, and other related information
This sentence "divulge othe interconnectedness of your system infrastructure" is a little odd. I would recommend something more direct and less fluffly, e.g. "Clicking on any of Middleware's interactive tables will produce popouts that walk you through each component involved in a request, how that component is behaving, and other related information
This sentence "divulge othe interconnectedness of your system infrastructure" is a little odd. I would recommend something more direct and less fluffly, e.g. "Clicking on any of Middleware's interactive tables will produce popouts that connect you to all information related to that data point that Middleware has collected. These popouts will allow you to drill down to root cause regardless of which data point you start from."